HACKING.md

```diff
@@ -11,6 +11,7 @@ To build EnMasse, you need
     * Node.js
     * npm
     * docker
+    * maven >=3.1
```

Comment:
out of curiosity, what's the need for a specific minimum version ?

Reply:
The maven requirement is from building a patched version of Artemis, and Artemis has a lower bound on 3.1 for maven versions.